lib/features/registration/pages/signup_page.dart

```diff
@@ -137,7 +137,14 @@ class _SignUpPageState extends State<SignUpPage> {
           builder: (context, state) {
             return isLoading
                 ? const Center(
-                    child: CircularProgressIndicator(),
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Text('Hold on, we are creating your account...'),
+                        SizedBox(height: 16),
+                        CircularProgressIndicator(),
```

Comment:
I started making it adaptive whenever I encounter this "CircularProgressIndicator.adaptive()"

Reply:
What does .adaptive() do? It just visually looks different?

Reply:
It's adapting the progress indicator to the platform. If iOS it display the cupertino progress indicator for Android is the Material one. We would have to do the same for dialogs as we use atm the cupertino dialog